### ISSUE TITLE
Windows Defender treat Tunic as Trojan:Win32/Fuery.B!cl

### ISSUE THREAD

**Issue by hohoaisan** [mikeslattery/tunic#13]
![image](https://user-images.githubusercontent.com/26185060/75646205-6e54b780-5c7b-11ea-9ebf-042df38c0825.png)

**Comment by mikeslattery** [comment]
This is a false positive.  There's only so much I can do about this.  Tunic installs a bootloader which can look suspicious to AV.

See also #7.

I have covered this in the [FAQ](https://github.com/mikeslattery/tunic/blob/master/doc/faq.md#when-i-download-tunicexe-it-disappears) and [security.md](https://github.com/mikeslattery/tunic/blob/master/doc/security.md) but I suppose I need to specifically warn about this in the Getting Started section.

I'll do the following:
* In "Getting Started", I'll describe what can be done if AV falsely detects as malware.
* In security.md, I'll go into more detail on how to audit tunic.exe to show it is not malware, to ease concerns of Tunic users.
* I'll investigage what Furey.B!cl is and see what pattern it's finding in Tunic.  This may be hard to do.
* Close this ticket.

I'm currently in the middle of a rewrite (#10) which may magically solve the issue.

I likely will _not_ be able to stop this from ever happening.  I can only help users to deal with it.

**Comment by mikeslattery** [comment]
I didn't meant to close.

**Comment by hohoaisan** [comment]
By adding certificates and valid publisher information to each release helps to correct this issue I wonder? Windows SmartScreen warns me about this as well.

**Comment by mikeslattery** [comment]
Yeah, you're right.  I'll look into [Certum's certs](http://www.certum.eu/certum/cert,offer_code_signing.xml), discounted for open source projects.

**Comment by mikeslattery** [comment]
Cheapest cert I could find is $153-$217 per year depending on many years I buy up front.  That's with an open source discount.  Commercial ones are much more.  Yet another reason why the world needs to get away from Windows.

**Comment by mikeslattery** [comment]
The OSS discounted Certum cert is $154 for the first year and $40/year after that.   Also, it's missing some features so I'm not 100% sure it would solve this problem.  Still by far the cheapest cert.
